tests/json_cart_parser_file_test.py

```diff
@@ -0,0 +1,210 @@
+import pytest
+import json
+import os
+from unittest.mock import mock_open, patch
+from src.parser.json_cart_parser import read_from_file, write_to_file, CartEncoder
+
+@pytest.fixture
+def valid_json_data_zero_items():
+    return '{"cart_name": "TestCart_0", "real_items": [], "virtual_items": [], "total": 0}'
+
+
+@pytest.mark.positive_read
+def test_read_from_file_zero_items(valid_json_data_zero_items):
+    with patch("builtins.open", mock_open(read_data=valid_json_data_zero_items)):
+        cart = read_from_file("test_file_path")
+        assert cart.cart_name == "TestCart_0"
+        assert cart.real_items == []
+        assert cart.virtual_items == []
+        assert cart.total == 0
+
+
+@pytest.mark.positive_write
+def test_write_to_file_zero_items(valid_json_data_zero_items):
+    test_file_path = "test_write_to_file_zero_items.txt"
+    test_file = open(test_file_path, "w+")
```

Comment:
```suggestion
    test_file = open(test_file_path, "w+")
    os.remove(test_file_path)
```

this part better add to fixture, read it here https://docs.pytest.org/en/6.2.x/fixture.html#requesting-fixtures
in Teardown/Cleanup (AKA Fixture finalization)

Reply:
ok, it is done